sdk/keyvault/Azure.Security.KeyVault.Certificates/src/Azure.Security.KeyVault.Certificates.csproj

```diff
@@ -5,7 +5,7 @@
     <AssemblyTitle>Microsoft Azure.Security.KeyVault.Certificates client library</AssemblyTitle>
     <Version>4.0.0-preview.7</Version>
     <PackageTags>Microsoft Azure Key Vault Certificates;$(PackageCommonTags)</PackageTags>
-    <PackageReleaseNotes>
+    <!--<PackageReleaseNotes>
```

Comment:
Are you planning to delete this, or are you trying to wait for freedback from keyvault?

Reply:
I'm releasing a new version today or tomorrow, so if this isn't ready I prefer preview.8 go out as-is.

Reply:
I wasn't sure what version those notes were for.

Reply:
Don't comment those out just yet. I haven't been able to release yet (waiting for sign-off on #9174).

Reply:
Am still testing to see how it all works on nuget. However you can add the notes to the changelog and it should be fine. With a link to CHANGELOG.md from nuget which is how all packages will be till I work on pulling out the appropriate release note.

Reply:
I've released preview 8, so you can just remove this section now.